extensions/notebook/src/book/remoteBookController.ts

```diff
@@ -14,6 +14,7 @@ const assetNameRE = /([a-zA-Z0-9]+)(?:-|_)([a-zA-Z0-9.]+)(?:-|_)([a-zA-Z0-9]+).(
 
 export class RemoteBookController {
 	constructor(public model: RemoteBookDialogModel, public outputChannel: vscode.OutputChannel) {
+		this.model.releases = [];
```

Comment:
Why not just have the model initialize releases itself? Right now the typing for the releases property is incorrect if whatever is creating it doesn't initialize the array - and there's not really any reason I can see not to just define the internal variable as `private _releases: IRelease[] = [];`

Reply:
that's a better idea. Let me do that.